### DISCUSSION TITLE
NotifyPlexToScan Disabled for Lidarr

### DISCUSSION THREAD

**Discussion by CurlyBorly** [RandomNinjaAtk/arr-scripts#73] [Q&A] [answered]
Is there a reason `NotifyPlexToScan` is commented out for the Lidarr Audio script? It seems like it's been this way since the  `lidarr-extended` repo based on the git blame.

https://github.com/RandomNinjaAtk/arr-scripts/blob/9e2b4f0774e2822a01d0aaaa79c0d00ecd741ecc/lidarr/Audio.service.bash#L716-L722

**Comment by RandomNinjaAtk** [comment, marked as answer]
Yes, because lidarr should be triggering the scan using the custom script, so it isn’t needed there….

**Reply by CurlyBorly** [reply]
So is that something that needs to be manually set up? I have been having to manually run the scanner in Plex for new albums to appear after they're imported to Lidarr

**Reply by RandomNinjaAtk** [reply]
If you enabled the auto config script, it should have added it to your lidarr web interface.

Lidarr web -> settings -> connect

**Reply by CurlyBorly** [reply]
Ah ok, I did not run the auto config script because I prefer my metadata profile setup, but I see you made some changes so I can disable that setup while allowing the others to run, thanks!

**Reply by RandomNinjaAtk** [reply]
Just fyi, I have not tested the changes…. But it should work….
